operators/dream_texture.py

```diff
@@ -105,7 +105,11 @@ def done_callback(future):
             result: ImageGenerationResult = future.result(last_only=True)
             for i, result_image in enumerate(result.images):
                 seed = result.seeds[i]
-                image = bpy_image(str(seed), result_image.shape[1], result_image.shape[0], result_image.ravel(), last_data_block)
+                prompt_string = context.scene.dream_textures_prompt.prompt_structure_token_subject
+                seed_str_length = len(str(seed))
+                trim_aware_name = (prompt_string[:58 - seed_str_length] + '..') if len(prompt_string) > 58 else prompt_string
+                name_with_trimmed_prompt = f"{trim_aware_name} ({seed})"
+                image = bpy_image(f"{name_with_trimmed_prompt} ({seed})", result_image.shape[1], result_image.shape[0], result_image.ravel(), last_data_block)
```

Comment:
Looks like this adds the seed to the name twice, but you can't see it in the result because it gets truncated 🙃 

```suggestion
                image = bpy_image(name_with_trimmed_prompt, result_image.shape[1], result_image.shape[0], result_image.ravel(), last_data_block)
```